app/helpers/carts_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CartsHelper
+  def menus
+    @menus ||= cart_items.joins(:menu).all
```

Comment:
that's tricky, `joins` performs INNER JOIN, so make sure you shouldn't show cart_items, that's doesn't have menu-relation. Maybe that's alrighty line, just FYI